api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderConstants.java

```diff
@@ -148,6 +148,7 @@ public static final class Apps {
     public static final class Extensions {
 
         public static final String CHECK_IN_VISIT_ACTION = "pih.checkin.visitAction";
+        public static final String TRIAGE_VISIT_ACTION = "pih.triage.visitAction";
```

Comment:
There are other triage_visits and this one seems specific for mch triage.  If you don't think it's needs, I'm fine with that @cioan

Reply:
Yes, agreed, that  including "MCOE" in the name males sense.

Reply:
Or perhaps "maternal" or "MCH" triage if we want something more generic?

Reply:
Thanks! I have renamed to MCH triage.